app/controllers/external_authors_controller.rb

```diff
@@ -76,7 +76,7 @@ def update
     end
     @invitation.mark_as_redeemed if @invitation && !params[:imported_stories].blank?
 
-    if @external_author.update_attributes(external_author_params[:external_author])
+    if @external_author.update_attributes(external_author_params[:external_author] || {})
```

Comment:
Sorry, I understand ( I missed the closing bracket so I thought it was associated with the if)